x/deployment/keeper/keeper_test.go

```diff
@@ -211,6 +211,53 @@ func Test_OnDeploymentClosed(t *testing.T) {
 	})
 }
 
+func Test_CloseGroup(t *testing.T) {
+	ctx, keeper := setupKeeper(t)
+	groups := createActiveDeployment(t, ctx, keeper)
+
+	t.Run("assert group 0 state closed", func(t *testing.T) {
+		assert.NoError(t, keeper.OnCloseGroup(ctx, groups[0]))
+		group, ok := keeper.GetGroup(ctx, groups[0].ID())
+		assert.True(t, ok)
+		assert.Equal(t, types.GroupClosed, group.State)
+
+		keeper.OnDeploymentClosed(ctx, group) // coverage: catch an additional code path
+		assert.Equal(t, types.GroupClosed, group.State)
+	})
+	t.Run("group 1 matched-state not-orderable", func(t *testing.T) {
+		group := groups[1]
+		assert.Equal(t, group.ValidateOrderable(), types.ErrGroupNotOpen, group.State.String())
```

Comment:
so, again: this is checking that it's not orderable, but it's not making sure that it's state isn't changing.

https://github.com/ovrclk/akash/pull/657#discussion_r445089848

I see the new test to make sure ancillary groups don't go from orderable -> not-orderable.  this one should be making sure that the state stays in `matched`.

You know what - forget it.  never mind.